### DISCUSSION TITLE
How to add top languages stat?

### DISCUSSION THREAD

**Discussion by Benhrpr** [alshedivat/al-folio#1560] [Q&A] [answered]
On the page with the trophies, commit info, etc. I was wondering how I could go about adding the top languages stat in addition to the general github stat that includes the number of commits, etc. Thanks.

**Comment by george-gca** [comment, marked as answer]
Check [my repo](https://github.com/george-gca/george-gca.github.io/tree/gca), I did exactly that. The only thing is that I don't know how it calculates the top languages part, since I don't believe my most used language is jupyter notebooks, but anyway you can get the idea.

**Reply by Benhrpr** [reply]
Btw...I saw you have a table of contents on your website. I saw you changed the code in cv.html but where else did you change it to determine tags 't' and 'cv'.

**Reply by george-gca** [reply]
Table of contents? You mean, in the cv page?

These changes were introduced in #1547, and there is [more to come](https://github.com/alshedivat/al-folio/issues/1551).

As of `t` and `cv` tags, this is part of the translations I made in the repository. Currently I keep a [multilingual fork of the al-folio](https://github.com/george-gca/multi-language-al-folio) repo, and my site is based on this template. I tend to make the increments I wish the template have in the original al-folio repo then keep the multilingual fork up-to-date with it.

To learn a little bit of how this works, check [my post](https://george-gca.github.io/blog/2022/dual-language-al-folio/) where I documented some of the process of creating the translated version. But all the needed changes are already made in the multilingual fork.

**Reply by Benhrpr** [reply]
Yes, the table of contents on the cv page. How would I go about adding this for a English only website?

**Reply by george-gca** [reply]
It is already done in this repo, they were introduced in #1547. Just update your repository with the new commits.
